frontend/src/googleMaps/LocateCurrentLocation.js

```diff
@@ -0,0 +1,30 @@
+
+import {Button} from "@material-ui/core";
+
+
+export default function LocateCurrentLocation ({panTo}) {
+
+    return (
+        <Button
+            variant="contained"
+            color="primary"
+                onClick={() => {
+            navigator.geolocation.getCurrentPosition(
+                (position) => {
+                    panTo({
+                        lat: position.coords.latitude,
+                        lng: position.coords.longitude,
+                    })
+                },
+                () => null);
```

Comment:
```suggestion
                });
```
do you need () => null